src/flexmock/api.py

```diff
@@ -311,75 +313,62 @@ def updated(self: Any) -> Any:
-                raised, instance = sys.exc_info()[:2]
-                assert raised, "no exception was raised"
-                message = "%s" % instance
-                expected = return_values[0].raises
-                if not expected:
-                    raise
-                args = return_values[0].value
-                expected_instance = expected(*args["kargs"], **args["kwargs"])
-                expected_message = "%s" % expected_instance
-                if inspect.isclass(expected):
-                    if expected is not raised and expected not in raised.__bases__:
-                        raise ExceptionClassError("expected %s, raised %s" % (expected, raised))
-                    if args["kargs"] and isinstance(args["kargs"][0], RE_TYPE):
-                        if not args["kargs"][0].search(message):
-                            raise (
-                                ExceptionMessageError(
-                                    'expected /%s/, raised "%s"'
-                                    % (args["kargs"][0].pattern, message)
-                                )
-                            )
-                    elif expected_message and expected_message != message:
+            return_values = expectation._return_values
+            if not return_values:
+                raise
+            raised, instance = sys.exc_info()[:2]
+            assert raised, "no exception was raised"
+            message = "%s" % instance
+            expected = return_values[0].raises
+            if not expected:
+                raise
+            args = return_values[0].value
+            expected_message = "%s" % expected(*args["kargs"], **args["kwargs"])
+            if inspect.isclass(expected):
+                if expected is not raised and expected not in raised.__bases__:
+                    raise ExceptionClassError("expected %s, raised %s" % (expected, raised))
+                if args["kargs"] and isinstance(args["kargs"][0], RE_TYPE):
+                    if not args["kargs"][0].search(message):
                         raise (
                             ExceptionMessageError(
-                                'expected "%s", raised "%s"' % (expected_message, message)
+                                'expected /%s/, raised "%s"' % (args["kargs"][0].pattern, message)
                             )
                         )
-                elif expected is not raised:
-                    raise ExceptionClassError('expected "%s", raised "%s"' % (expected, raised))
-            else:
-                raise
+                elif expected_message and expected_message != message:
+                    raise (
+                        ExceptionMessageError(
+                            'expected "%s", raised "%s"' % (expected_message, message)
+                        )
+                    )
+            elif expected is not raised:
+                raise ExceptionClassError('expected "%s", raised "%s"' % (expected, raised))
```

Comment:
I prefer having single quotes inside the message :stuck_out_tongue: 
```suggestion
            elif expected is not raised:
                raise ExceptionClassError(f"expected '{expected}', raised '{raised}'")
```

Reply:
Same as the f-string suggestion, let's modify all the quotes in a different unit of work and ensure standardization with the linter/formatter.